src/components/node.js

```diff
@@ -100,7 +100,7 @@ function Node({
   onNodeMouseLeave = () => {},
   onNodeMove = () => {},
   onNodeSelected = () => {},
-  onNodeUpdate = () => {},
+  onNodeUpdate = () => Promise.resolve(),
```

Comment:
We assign a resolved promise to onNodeUpdate in case someone forgot to assign a onNodeUpdate function (I don't think that could ever happen anymore).